integrations/influxdb_connector/influxdb-timestream-connector/template.yml

```diff
@@ -566,6 +566,19 @@ Outputs:
   Endpoint:
     Description: The endpoint for the REST API Gateway.
     Value: !Sub https://${RestApiGateway}.execute-api.${AWS::Region}.amazonaws.com/${RestApiGatewayStageName}
+  ExecutionPolicy:
+    Description: The IAM policy permissions required for invoking the InfluxDB Timestream Connector.
```

Comment:
Nice, now users will have permissions clearly outlined, and it's not too verbose.